test/user_test.rb

```diff
@@ -36,4 +37,21 @@ def test_finding_a_specific_user
     assert_equal user.full_name, 'Developer'
     assert_equal user.email, 'developer@artemisag.com'
   end
+
+  def test_finding_a_user_with_included_organizations
+    stub_request(:get, 'http://localhost:3000/api/v3/facilities/2/users/42?include=organizations')
+      .to_return(body: {data:
+                         {id: '42',
+                          type: 'users',
+                          attributes: {id: 42, full_name: 'Developer', email: 'developer@artemisag.com'}},
+                        included: [{id: '1', type: 'organizations', attributes: {id: 1, name: 'Vegetable Sky'}}]}.to_json)
+
+    user = ArtemisApi::User.find(42, 2, @client, include: "organizations")
+    assert_equal user.full_name, 'Developer'
+    assert_equal user.email, 'developer@artemisag.com'
+    assert_equal @client.objects['organizations'].count, 1
+
+    organization = ArtemisApi::Organization.find(1, @client)
```

Comment:
when the organization is included and stored in an object like this, that means the next request will use the stored object and thus won't have to make a separate API call right? it would be great if that was explicitly stated in the test - either in the test name, as a separate test case, or with a comment.